test/files/neg/inductive-implicits3.scala

```diff
@@ -0,0 +1,196 @@
+object shapeless {
```

Comment:
(puts on RMS harddrive / hat) you may wish to include the shapeless LICENSE / NOTICE, unless this is a subset of which you are the sole author.